Docs

michaelquinn32 opened the pull request:
The goals of this process were to increase doc visibility by combining
separate files, provide better cross references and surface aspects
of `skim` that might not be well known. I also tried to replace all
Rd style formatting with roxygen's new support for markdown.

This commit also fixes the vignettes, which weren't accessible within
R.

elinw commented:
Also can you explain the vignette comment? They are reachable from the index page as long as you do a install.packages(), if you install from github etc they don't get built which is part of how devtools works (I think there is possibly a way to force it).  We confirmed this the other day in a closed issue in the issue tracker.

michaelquinn32 commented:
Sure!

I moved quiet_impl to wide.R

I ran both of those functions. Spell check mentioned some abbreviations, which I cleaned up. I also missed some Rd formatting from before. GP only mentioned code coverage and long lines.

I ran devtools::build_vignette, which showed that we were keeping files in inst/docs, instead of the correct inst/doc. When I installed locally, the package index now correctly links to the vignettes. I didn't realize that devtools wants you to build your vignettes with a separate function.

As far as the comment on accessibility standards, I'm sorry but I'm not sure what you're referring to. What should I change?

elinw commented:
That's awesome about docs versus doc ... 
So a list in html should not use `<p></p>` tags it should use <ul><li>abc</li><li>def</li></ul>  so that a screen reader knows it is a list of items.  But anyway that's something I can deal with later since we aren't doing it now anyway (and I was also thinking how nice to have a refactor that really doesn't break the tests).

So this looks good to me.
